.circleci/config.yml

```diff
@@ -977,6 +977,9 @@ jobs:
     steps:
       - run: git config --global core.autocrlf false
       - checkout
+      - run:
+          name: Install dependencies
+          command: python -m pip install --user requests
```

Comment:
Do we still need this?

Reply:
Yes. This is unrelated to the symlink - in fact, this was the original issue I encountered; I only realized the unavailability of `python3` *after* this was fixed.